Downloads.html

```diff
@@ -52,7 +53,7 @@
                 </div>
                 <div class="col-md-4">
                     <h2>Quick Start Guide</h2>
-                    <p>The DDF install guide and documentation are located on the wiki site.</p>
+                    <p>Get a sample or test installation up quickly.</p>

```

Comment:
I don't have any suggestions, but I found myself having to read this phrase a couple times in order to understand it. Thoughts?